patterns/base/p/sass/partials/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%base--p {
+  
```

Comment:
An empty placeholder selector yield no code. So referencing this empty placeholder in p.scss will error out.